spec/spec_helper.rb

```diff
@@ -11,6 +11,7 @@
 
 
 require 'rspec/rails'
+require 'devise'
```

Comment:
My understanding is that anything inthe Gemfile is auto-included, things in the gemspec are not, hence our support files are littered with `require`s.